src/otx/algo/hooks/recording_forward_hook.py

```diff
@@ -21,7 +21,7 @@ class BaseRecordingForwardHook:
         normalize (bool): Whether to normalize the resulting saliency maps.
     """
 
-    def __init__(self, head_forward_fn: Callable, normalize: bool = True) -> None:
+    def __init__(self, head_forward_fn: Callable = lambda x: x, normalize: bool = True) -> None:
```

Comment:
Does this new default value make sense, or we can use None?
If we use identify for act map hook, passing identify to the constructor there would be more clear

Reply:
Indeed, none will work just for now. Fixed